src/app/src/components/DownloadFacilitiesButton.jsx

```diff
@@ -19,6 +19,7 @@ const downloadFacilitiesStyles = theme =>
             height: '45px',
             margin: '5px 0',
             backgroundColor: theme.palette.action.main,
+            color: theme.palette.getContrastText(theme.palette.action.main),
```

Comment:
We have the calculate the contrast text for 'action'; as a custom text set, it doesn't get calculated automatically